tests/fixtures/fixtures.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+import tensorflow as tf
+
+import garage.misc.logger as logger
+from garage.misc.tensorboard_output import TensorBoardOutput
+
+
+class GarageTestCase(unittest.TestCase):
+    def setUp(self):
+        self.sess = tf.Session(graph=tf.Graph())
+        self.sess.__enter__()
+        logger._tensorboard = TensorBoardOutput()
+
+    def tearDown(self):
```

Comment:
I had trouble with this crashing when I did this, so glad that this worked for you lol.